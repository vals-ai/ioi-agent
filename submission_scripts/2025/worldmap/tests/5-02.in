version https://git-lfs.github.com/spec/v1
oid sha256:7c70d351fb3caa3bada734fff3f9b460f9d5bcf960c039b3a620fd7514a80591
size 5631
