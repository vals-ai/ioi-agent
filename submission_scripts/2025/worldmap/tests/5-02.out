version https://git-lfs.github.com/spec/v1
oid sha256:7cec72ca4178ad1c60f2eeb9d09dbbe62585c3c1762fc05ccab4afdcf6abdf73
size 46175
